src/components/Authentication/SignUp.js

```diff
@@ -57,11 +57,23 @@ const SignUpFormBase = props => {
     props.firebase
       .doCreateUserWithEmailAndPassword(form.email, form.passwordOne)
       .then(authUser => {
+        authUser.user
+          .updateProfile({
+            displayName: form.username,
+            photoURL:
+              'https://firebasestorage.googleapis.com/v0/b/bookio.appspot.com/o/images%2Frobot.png?alt=media&token=d0cc8a48-aaad-4e3b-9c08-3e341ed8165e',
+          })
+          .then(function() {
```

Comment:
Same here, please add some proper error handling.

Reply:
Just according to the standard, we should catch the error, but there are no errors now. So I don't know what to catch.